diff --git a/node_modules/cui-llama.rn/cpp/ggml-opencl/kernels/embed_kernel.py b/node_modules/cui-llama.rn/cpp/ggml-opencl/kernels/embed_kernel.py
--- a/node_modules/cui-llama.rn/cpp/ggml-opencl/kernels/embed_kernel.py
+++ b/node_modules/cui-llama.rn/cpp/ggml-opencl/kernels/embed_kernel.py
@@ -12,8 +12,8 @@
         logger.info("Usage: python embed_kernel.py <input_file> <output_file>")
         sys.exit(1)
 
-    ifile = open(sys.argv[1], "r")
-    ofile = open(sys.argv[2], "w")
+    ifile = open(sys.argv[1], "r", encoding="utf-8")
+    ofile = open(sys.argv[2], "w", encoding="utf-8", newline="\n")
 
     for i in ifile:
         ofile.write('R"({})"\n'.format(i))
