diff --git a/node_modules/react-native-mmkv/android/src/main/java/com/margelo/nitro/mmkv/NitroMmkvPackage.java b/node_modules/react-native-mmkv/android/src/main/java/com/margelo/nitro/mmkv/NitroMmkvPackage.java
index e6993ab..9ba2dc8 100644
--- a/node_modules/react-native-mmkv/android/src/main/java/com/margelo/nitro/mmkv/NitroMmkvPackage.java
+++ b/node_modules/react-native-mmkv/android/src/main/java/com/margelo/nitro/mmkv/NitroMmkvPackage.java
@@ -28,6 +28,7 @@ public class NitroMmkvPackage extends TurboReactPackage {
   }
 
   static {
-    NitroMmkvOnLoad.initializeNative();
+      Class<?> onLoad = Class.forName("com.margelo.nitro.mmkv.NitroMmkvOnLoad");
+      onLoad.getMethod("initializeNative").invoke(null);
   }
 }
